expression/builtin_miscellaneous_test.go

```diff
@@ -15,10 +15,47 @@ package expression
 import (
 	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/ast"
+	"github.com/pingcap/tidb/util/testleak"
 	"github.com/pingcap/tidb/util/testutil"
 	"github.com/pingcap/tidb/util/types"
 )
 
+func (s *testEvaluatorSuite) TestInetAton(c *C) {
+	defer testleak.AfterTest(c)()
+	tbl := []struct {
+		Input    interface{}
+		Expected interface{}
+	}{
+		{"", nil},
+		{nil, nil},
+		{"255.255.255.255", 4294967295},
+		{"0.0.0.0", 0},
+		{"127.0.0.1", 2130706433},
+		{"0.0.0.256", nil},
+		{"113.14.22.3", 1896748547},
+		{"127", 127},
+		{"127.255", 2130706687},
+		{"127,256", nil},
+		{"127.2.1", 2130837505},
+		{"123.2.1.", nil},
+		{"127.0.0.1.1", 545460846849}, // test in MySQL, it not returns NULL.
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6143402/24185435/1d7b8702-0f0e-11e7-9bc5-2b83e4ca5a8c.png)

Oh..
MySQL5.6 and MySQL5.7 is different...I test in MySQL 5.6 before. 
I will fixed it tonight…